karate-tests/src/test/java/patients/patientAccess/getCoverageDetails.feature

```diff
@@ -0,0 +1,78 @@
+@no-oas
+Feature: Patient Access (Retrieve)
+    Retrieve a chargeable snippet for a patient
+
+Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    * json patientCoverageResultEntry  = karate.readAsString('classpath:schemas/searchSchemas/patientCoverageResultEntry.json')
+    * json coverageBundle = karate.readAsString('classpath:schemas/searchSchemas/patientCoverageBundle.json')
+    
+    * configure url = baseURL
+
+ @sandbox
+   Scenario: Happy path - Retrieve patient coverage details
+    * def p9number = '9733162868'
+    * def accessToken = karate.call('classpath:auth/auth-redirect.feature', {userID: p9number, scope: 'nhs-login'}).accessToken
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * configure headers = requestHeaders
+    * path 'Coverage'
+    * param beneficiary:identifier = p9number
+    * method get
+    * status 200
+    * match response == coverageBundle
+    * match response.entry[0].resource.status == 'active'
+    * match response.entry[0].resource.beneficiary.identifier.value == p9number
+    * match response.entry[0].resource.identifier[0].assigner contains { display: '#notnull'}
+
+  @sandbox
+ Scenario: Happy path - patient has no coverage details
+    * def P9WithNoCoverage = '9733162876'
+    * def accessToken = karate.call('classpath:auth/auth-redirect.feature', {userID: P9WithNoCoverage, scope: 'nhs-login'}).accessToken
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * configure headers = requestHeaders
+    * path 'Coverage'
+    * param beneficiary:identifier = P9WithNoCoverage
+    * method get
+    * status 200
+    * match response.entry[0] == '#notpresent'
```

Comment:
Might want to check that we still get the other data returned from this response such as meta.versionId and ETag header

Reply:
Thanks for highlighting this validation, I have included meta. versionID and Etag header validations